app/services/order_purchaser.rb

```diff
@@ -55,15 +55,39 @@ def order_detail_updater
   end
 
   def order_in_the_past!
-    unless order.can_backdate_order_details?
+    unless can_backdate_order_details?
```

Comment:
Weird, my codeclimate plugin is telling me `Favor modifier `unless` usage when having a single-line body.`, but I don't see i in codeclimate itself.

Reply:
I feel like if we made that change the line length would be a bit unwieldy.